landoapi/spec/swagger.yml

```diff
@@ -135,6 +135,11 @@ paths:
         Sends request to transplant service and responds with just the
         status code
       parameters:
+        - name: API-Key
```

Comment:
For Transplant it's just its own API-Key. We certainly make it more descriptive, just wondering if it's needed.